app/src/main/javascript/bundles/experiment-page/src/TSnePlotViewRoute.js

```diff
@@ -166,16 +165,14 @@ class TSnePlotViewRoute extends React.Component {
           wrapperClassName={`row expanded`}
           ks={ks}
           selectedClusterByCategory={search.cellGroupType || search.k || preferredK}
-          selectedClusterId={this.state.selectedClusterId}
-          onChangeClusterId={(colourByCategory, colourByValue) => {
+          selectedK={this.state.selectedClusterId}
+          onSelectK={(colourByValue,colourByCategory) => {
```

Comment:
The parameter `colourByCategory` isn’t passed and it’s also not in the function signature. Let’s remove it. I guess it’s a leftover from the previous `onChangeClusterId`.